tools/crm_mon_print.c

```diff
@@ -1014,9 +1027,20 @@ print_xml_status(pcmk__output_t *out, pe_working_set_t *data_set,
 
     /*** NODES ***/
     if (is_set(show, mon_show_nodes)) {
-        out->begin_list(out, NULL, NULL, "nodes");
+        gboolean printed_header = FALSE;
+
```

Comment:
I'm thinking we should begin the list regardless for XML. An empty node list (vs none) is potentially meaningful for something parsing XML.